File: src/my_project_typescript/easy_problems_typescript/from51To100/threeConsecutiveOdd.ts
function threeConsecutiveOdds(arr: number[]): boolean {

    let count: number = 0
    for (let num of arr){
        if (num%2 === 0){
            count = 0
        } else {
            count += 1
        }
        if (count >=3){
            
            return true
            
        }
    }

    return false 
};



console.log(threeConsecutiveOdds([2,6,4,1]))

console.log(threeConsecutiveOdds([1,2,34,3,4,5,7,23,12]))

File: src/my_project/easy_problems/from151To200/three_consecutive_odd.py
from typing import List 

class Solution:
    def threeConsecutiveOdds(self, arr: List[int]) -> bool:

        count: int = 0

        for num in arr:

            if num % 2 == 0:
                count = 0
            else: 
                count += 1
            if count >= 3:
                return True

        return False 

solution = Solution()

print(solution.threeConsecutiveOdds(arr = [2,6,4,1]))

print(solution.threeConsecutiveOdds(arr = [1,2,34,3,4,5,7,23,12]))
